gammapy/data/observations.py

```diff
@@ -356,6 +361,32 @@ def check(self, checks="all"):
         checker = ObservationChecker(self)
         return checker.run(checks=checks)
 
+    def _create_missing_gti(self):
+        """Returns a GTI table containing TSTART and TSTOP from the obs table meta data.
+
+        This method can be removed once GTI tables are explicitly required in Gammapy.
+        """
+        header = OrderedDict(
+            EXTNAME="GTI",
+            HDUCLASS="GADF",
+            HDUDOC="https://github.com/open-gamma-ray-astro/gamma-astro-data-formats",
+            HDUVERS="0.2",
+            HDUCLAS1="GTI",
+            MJDREFI=self.data_store.obs_table.meta["MJDREFI"],
```

Comment:
We already use `time_ref_from_dict(self.data_store.obs_table.meta)` in other places in this file. Suggest to use it here also to get consistent behaviour.

Reply:
`time_ref_from_dict` already returns a `Time` object, whereas here i rather need the int and float values.